components/movementsensor/gpsrtkpmtk/gpsrtkpmtk.go

```diff
@@ -326,8 +327,13 @@ func (g *rtkI2C) getStream(mountPoint string, maxAttempts int) error {
 	}
 
 	if err != nil {
-		g.logger.Errorf("Can't connect to NTRIP stream: %s", err)
-		return err
+		// if the error is related to ICY, we log it as warning.
+		if strings.Contains(err.Error(), "ICY") {
+			g.logger.Warnf("Can't connect to Ntrip Steam: %s", err)
```

Comment:
```suggestion
			g.logger.Warnf("Can't connect to NTRIP Steam: %s", err)
```

Reply:
But it is connecting to the stream from what I understand, just spamming the HTTPS error?

Reply:
My understanding is that some message response looks like  `ICY 200 OK` instead of `HTTP/1.1 200 OK`.  but the ntrip client library that we use gets hangup on the "ICY" part since it only supports HTTP/1.1 and 2. So yes, we do connect to the stream after a few attempt with the `malformed HTTP version "ICY"` error.

Reply:
So I'd change the first part of the string to something like: "attempting to connect to old HTTPS protocol: err", so it's not telling you that it's not connecting to the ntrip when it's giving you another error or warning.